src/mumble/AudioWizard.cpp

```diff
@@ -638,6 +638,9 @@ void AudioWizard::on_skwPTT_keySet(bool valid, bool last) {
 
 void AudioWizard::on_qcbEcho_clicked(bool on) {
 	g.s.bEcho = on;
+	if (g.s.bEcho) {
+		g.s.iEchoOption = 1;  // ECHO_CANCEL_DEFAULT = 1
```

Comment:
Why not make this explicit?
```suggestion
		g.s.iEchoOption = ECHO_CANCEL_DEFAULT;
```